packages/accounts/src/rpc-api/getAccountInfo.ts

```diff
@@ -81,6 +129,12 @@ export type GetAccountInfoApi = {
                 encoding: 'base58';
             }>,
     ): GetAccountInfoApiResponseBase & NestInRpcResponseOrNull<AccountInfoWithBase58EncodedData>;
+    /**
+     * Fetches information associated with the account at the given address.
+     *
+     * {@label base58-legacy}
+     * @see https://solana.com/docs/rpc/http/getaccountinfo
+     */
```

Comment:
I think it'd make sense to copy the base58 info here (as you did in the equivalent rpc-api file)

Reply:
![Added via Giphy](https://media3.giphy.com/media/v1.Y2lkPWM5NDg3NzQzZGw5eHlkaWFwMTN0ZDBjc2k3N2RmbjB1cXJycXE3MTY3ZjlvZGhicyZlcD12MV9naWZzX3NlYXJjaCZjdD1n/3og0IEitjqvkkD5i5W/giphy.gif)